Java/libraries/recognizers-text-sequence/src/main/java/com/microsoft/recognizers/text/sequence/extractors/BaseSequenceExtractor.java

```diff
@@ -75,15 +75,14 @@ protected String getExtractType() {
                     String substr = text.substring(start, start + length);
                     Function<Match, Boolean> matchFunc = match -> match.index == start && match.length == length;
 
-                    Stream<Object> matchStream = Arrays.stream(matchSource.entrySet().toArray());
-                    if (matchStream.anyMatch(o -> matchFunc.apply((Match)o))) {
+                    if (matchSource.keySet().stream().anyMatch(o -> matchFunc.apply(o))) {
                         Match srcMatch = (Match)matchSource.keySet().toArray()[0];
                         ExtractResult extResult = new ExtractResult();
 
                         extResult.setStart(start);
                         extResult.setLength(length);
                         extResult.setText(substr);
-                        extResult.setType(this.extractType);
+                        extResult.setType(this.getExtractType());
```

Comment:
here we change the access to the variable to access through its getter. Is it ok that we can access both the variable through the getter and directly? Generally, if there is a getter we shouldn't be able to access the variable directly, but I don't know about Java

Reply:
Same

Reply:
+1 I'll double check

Reply:
Yep, that's correct, I rollbacked this. I'll push it asap